dist/documentation/schemas/woosmap_http_schema_assetrequest.html

```diff
@@ -180,10 +180,13 @@ <h4 id="AssetRequest-types" class="add-link schema-object-property-key">
         </h4>
       </td>
       <td>optional</td>
-      <td>object</td>
+      <td></td>
```

Comment:
why remove `object` in cell here? is empty cell really wanted?

Reply:
This is automatically generated and for now it's not used anywhere.

Reply:
The use of OneOf for user_properties type cause this. Without using OneOf, the schema is not validated using examples validator because children properties are unknown. This userProperties is of an unknown schema so not obvious to describe it.